src/lib/stores/schedules.ts

```diff
@@ -204,6 +205,18 @@ export const submitEditSchedule = async (
     },
   };
 
+  const fields = body.schedule.action.startWorkflow?.header?.fields;
+  if (fields && Object.keys(fields).length > 0) {
+    const entries = Object.entries(fields);
+    for (const [key, value] of entries) {
+      const encodedValue = await encodePayloads(
+        stringifyWithBigInt(value),
+        'json/plain',
```

Comment:
Do we want to use the `encoding` value here as well? 

```suggestion
        encoding
```

Reply:
I think we just use plain/json for now. There's no UI for editing headers for schedules and not sure we want to add that yet. If we need to we can, but headers are almost always json/plain.

Reply:
Yeah only concern here is that if a user has Codec Server set up with users who don't have `/encode` permissions then it will error on save unless we do a similar approach to https://github.com/temporalio/ui/pull/2384.

Reply:
👍 I added a try/catch for now and return if encoding errors for header fields